resources/views/components/code.blade.php

```diff
@@ -12,7 +12,11 @@
                         {{ $letters[strtolower($character)] ?? $character }}
                     </span>
 
-                    <div class="bg-gray-200 w-12 h-12"></div>
+                    <div class="
+                    @if(preg_match("/[a-z]/i", $character))
+                        bg-gray-200
+                    @endif
+                         w-12 h-12"></div>
```

Comment:
This would look a bit better.

```suggestion
                    <div 
                        @class([
                            'w-12 h-12',
                            'bg-gray-200' => preg_match("/[a-z]/i", $character),
                        ])
                    ></div>
```